libdlib/CMakeLists.txt

```diff
@@ -36,25 +27,25 @@ include_directories(
 	common/include
 )
 
-foreach(library ${libraries})
-	add_library(${library} SHARED)
-	add_dependencies(${library} copy_${library})
-	add_dependencies(copy_${library} build_dlib187)
-	set_target_properties(${library} PROPERTIES LINKER_LANGUAGE CXX)
-endforeach()
+add_library(dlib SHARED)
+add_dependencies(dlib build_dlib187)
+set_target_properties(dlib PROPERTIES LINKER_LANGUAGE CXX)
 set_target_properties(build_dlib187 PROPERTIES LINKER_LANGUAGE CXX)
 
 
 #############
 ## Install ##
 #############
-## Mark executables and/or libraries for installation
 install(TARGETS dlib
 	ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
 	LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-	RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+	RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
```

Comment:
I'm not sure, regarding http://docs.ros.org/indigo/api/catkin/html/howto/format2/building_libraries.html it is ${..._GLOBAL_BIN_...}, also done this way in libntcan or libphidgets